cdap-ui/app/cdap/services/WizardStores/AddNamespace/AddNamespaceStore.js

```diff
@@ -225,7 +238,8 @@ const createAddNamespaceStore = () => {
       security,
       preferences
     }),
-    defaultInitialState
+    defaultInitialState,
+    composeEnhancers()
```

Comment:
Why do we need this if we are not applying any middleware here?

Reply:
So the redux extension returns a redux enhancer which can be composed with other middlewares .. if the user who is using the UI doesn't have redux extension installed in the browser then it will just be an empty compose which will be ignored by redux. [Redux extension reference](https://github.com/zalmoxisus/redux-devtools-extension#12-advanced-store-setup)